cinema/views.py

```diff
@@ -87,6 +115,8 @@ class MovieSessionViewSet(viewsets.ModelViewSet):
         )
     )
     serializer_class = MovieSessionSerializer
+    authentication_classes = (TokenAuthentication,)
+    permission_classes = (IsAdminOrIfAuthenticatedReadOnly,)
```

Comment:
The `MovieSessionViewSet` uses `IsAdminOrIfAuthenticatedReadOnly` for permission classes, while the `OrderViewSet` uses `IsAuthenticated`. Verify that these permissions align with the intended access control policies for each viewset.